mongorestore/options.go

```diff
@@ -67,23 +67,24 @@ func (*InputOptions) Name() string {
 
 // OutputOptions command line argument long names
 const (
-	DropOption                     = "--drop"
-	DryRunOption                   = "--dryRun"
-	WriteConcernOption             = "--writeConcern"
-	NoIndexRestoreOption           = "--noIndexRestore"
-	ConvertLegacyIndexesOption     = "--convertLegacyIndexes"
-	NoOptionsRestoreOption         = "--noOptionsRestore"
-	KeepIndexVersionOption         = "--keepIndexVersion"
-	MaintainInsertionOrderOption   = "--maintainInsertionOrder"
-	NumParallelCollectionsOption   = "--numParallelCollections"
-	NumInsertionWorkersOption      = "--numInsertionWorkersPerCollection"
-	StopOnErrorOption              = "--stopOnError"
-	BypassDocumentValidationOption = "--bypassDocumentValidation"
-	PreserveUUIDOption             = "--preserveUUID"
-	TempUsersCollOption            = "--tempUsersColl"
-	TempRolesCollOption            = "--tempRolesColl"
-	BulkBufferSizeOption           = "--batchSize"
-	FixDottedHashedIndexesOption   = "--fixDottedHashIndex"
+	DropOption                       = "--drop"
+	DryRunOption                     = "--dryRun"
+	WriteConcernOption               = "--writeConcern"
+	NoIndexRestoreOption             = "--noIndexRestore"
+	ConvertLegacyIndexesOption       = "--convertLegacyIndexes"
+	NoOptionsRestoreOption           = "--noOptionsRestore"
+	KeepIndexVersionOption           = "--keepIndexVersion"
+	MaintainInsertionOrderOption     = "--maintainInsertionOrder"
+	NumParallelCollectionsOption     = "--numParallelCollections"
+	NumInsertionWorkersOption        = "--numInsertionWorkersPerCollection"
+	StopOnErrorOption                = "--stopOnError"
+	BypassDocumentValidationOption   = "--bypassDocumentValidation"
+	PreserveUUIDOption               = "--preserveUUID"
+	TempUsersCollOption              = "--tempUsersColl"
+	TempRolesCollOption              = "--tempRolesColl"
+	BulkBufferSizeOption             = "--batchSize"
+	FixDottedHashedIndexesOption     = "--fixDottedHashIndex"
+	FixDuplicatedLegacyIndexesOption = "--fixDuplicatedLegacyIndexes"
```

Comment:
I may be recalling an outdated discussion, but I thought when we last discussed we were not going to add a new flag and instead make it part of the default `--convertLegacyIndexes` behavior

Reply:
I may remember incorrectly, @tfogo Do you remembre if we would need a flag or not?

Reply:
I think we agreed on not having a flag but adding a note to the documentation.

Reply:
ah, cool. I will remove the flag then. Thanks!